File: 250719/친근하지 않은 수/unfriendly-number.py
# 입력
intN = int(input())

# 변수 정의
intCount = 0
# 반복 및 조건문
for i in range(1, intN + 1, 1) :
    if i % 2 == 0 or i % 3 == 0 or i % 5 == 0 :
        continue
    else :
        intCount += 1

# 출력
print(intCount)

File: 250719/친근하지 않은 수/unfriendly-number.java
// import
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        // 입력
        Scanner sc = new Scanner(System.in);
        int intN = sc.nextInt();

        // 변수
        int intCount = 0;

        // 반복 및 조건문
        for (int i = 1; i < intN + 1; i++) {
            if (i % 2 == 0 || i % 3 == 0 || i % 5 == 0) {
                continue;
            } else {
                intCount += 1;
            }
        }

        // 출력
        System.out.println(intCount);
    }
}